tests/conftest.py

```diff
@@ -1091,6 +1091,21 @@ def log_cli_level(pytestconfig):
     return pytestconfig.getini('log_cli_level') or 'DEBUG'
 
 
+TEMP_FILE = (
```

Comment:
do we really need to have the file for sharing the information? Cannot we put this information to the configuration?

I think you can use config.RUN['load_status'] instead of opening file / reading / writing.
I think it would be easier to do through this shared config.

As you are using multiThreading

Reply:
will be addressed with https://github.com/red-hat-storage/ocs-ci/issues/2699

Reply:
Will be fixed here:  https://github.com/red-hat-storage/ocs-ci/issues/2699